modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
 * Outbound Operations
 * Outbound Protocol
 * Outbound Endpoints
```

Comment:
Are we consistent here with the use of "Outbound - " prefix?

There are 4 charts: Ports, Operations, Protocols, and Endpoints

"Ports" is missing here.

Reply:
```suggestion
* Outbound Endpoints
* Outbound Ports
```